itests/org.jboss.tools.openshift.ui.bot.test/resources/osProjectWithResources/.project

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>os3projectWithResources</name>
+	<name>osProjectWithResources</name>
```

Comment:
Seems like you want to keep .project files around, right?

Reply:
Exactly :)